webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -18,10 +19,22 @@ import { DEFAULT_INTERVAL } from './constants';
 
 const interval = process.env.NOTIFICATIONS_POLLING || DEFAULT_INTERVAL;
 
-const getNotifications = url => get({ key: NOTIFICATIONS, url });
+const handleNotificationPollingError = (error, stopNotificationPolling) => {
+  stopNotificationPolling();
```

Comment:
This will mean notifications will stop after one unsuccessful poll. Is it intended? Basically @ekohl's [comment](https://github.com/theforeman/foreman/pull/7718#pullrequestreview-423652334) is still valid for notifications.

Reply:
thanks, moved it into the `if` context